cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -321,9 +321,9 @@ The deprecated built-in procedures for listing constraints, such as `db.constrai
 
 ==== Listing constraints examples
 
-include::listing-all-constraints.asciidoc[leveloffset=+2]
+include::listing-all-constraints.asciidoc[leveloffset=+3]
```

Comment:
You added them as `+4` for indexes and `+3` for constraints, is that intended?

Reply:
Thanks @Hunterness - yes, shifting this number down one level drops the sub-header down. It will be different on different pages, depending on how nested the header tags are.